include/veriblock/abfi/bitcoin/serialize.hpp

```diff
@@ -0,0 +1,1016 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
+// Copyright (c) 2009-2018 The Bitcoin Core developers
+// Copyright (c) 2019-2021 Xenios SEZC
+// https://www.veriblock.org
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_SERIALIZE_H
+#define BITCOIN_SERIALIZE_H
+
+#include <stdint.h>
+#include <string.h>
+
+#include <algorithm>
+#include <ios>
+#include <limits>
+#include <map>
+#include <memory>
+#include <set>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "veriblock/pop/blob.hpp"
+#include "veriblock/pop/read_stream.hpp"
+#include "veriblock/pop/slice.hpp"
+#include "veriblock/pop/write_stream.hpp"
```

Comment:
```suggestion
#include <veriblock/pop/write_stream.hpp>
```
Pls do same for other includes

Reply:
fixed